### PR TITLE
Prepare for official release

### PR DESCRIPTION
**Pull request by AndrewMusser**
Here's what's being done:

- [x] Make it a general omni.kit extension by removing all dependencies on `omni.isaac`. 
- [x] Revisit app categorization.
- [x] Finalize docs. 
- [ ] Publish the extension.

### CODE REVIEW

**Review by Joshpolansky — APPROVED** [review]

File: README.md [hunk 1]
```diff
@@ -6,24 +6,26 @@ info@loupe.team
 
 # Description
 
-Beckhoff Bridge Extension for Isaac Sim. Developed using [pyads](https://github.com/stlehmann/pyads).
+This is an extension that connects Beckhoff PLCs into the Omniverse ecosystem. It leverages [pyads](https://github.com/stlehmann/pyads) to set up an ADS client for communicating with the PLC.
```
Comment (shanereetz) [inline README.md:9]:
```suggestion
This is an extension that connects Beckhoff PLCs into the Omniverse ecosystem. It leverages [pyads](https://github.com/stlehmann/pyads) to set up an ADS client for communicating with PLCs. 
```

File: exts/loupe.beckhoff_bridge/config/extension.toml [hunk 2]
```diff
@@ -4,12 +4,12 @@ order = 0
...
 version = "0.1.0"
-category = "Connector"
+category = "simulation"
 title = "Beckhoff Bridge"
 description = "A bridge for connecting Omniverse to Beckhoff PLCs over ADS"
 authors = ["Loupe"]
-repository = "https://github.com/loupeteam/IsaacSim_Beckhoff_Bridge_Extension"
-keywords = []
+repository = "https://github.com/loupeteam/Omniverse_Beckhoff_Bridge_Extension"
+keywords = ["Beckhoff", "Digital Twin", "ADS"]
```
Comment (shanereetz) [inline exts/loupe.beckhoff_bridge/config/extension.toml:12]:
```suggestion
keywords = ["Beckhoff", "Digital Twin", "ADS", "PLC"]
```

File: exts/loupe.beckhoff_bridge/config/extension.toml [hunk 3]
```diff
@@ -18,12 +18,11 @@ icon = "data/icon.png"
...
 
 [python.pipapi]
 requirements = ['pyads']
 use_online_index = true
 
 [[python.module]]
-name = "loupe.beckhoff_bridge"
\ No newline at end of file
+name = "loupe.beckhoff_bridge"
+public = true
```
Comment (shanereetz) [inline exts/loupe.beckhoff_bridge/config/extension.toml:28]: 🎉

File: exts/loupe.beckhoff_bridge/docs/README.md [hunk 4]
```diff
@@ -0,0 +1,77 @@
+# Beckhoff Bridge
+
+The Beckhoff Bridge is an [NVIDIA Omniverse](https://www.nvidia.com/en-us/omniverse/) extension for communicating with [Beckhoff PLCs](https://www.beckhoff.com/en-en/) using the [ADS protocol](https://infosys.beckhoff.com/english.php?content=../content/1033/cx8190_hw/5091854987.html&id=).
+
+# Installation
+
+### Install from registry
+
+This is the preferred method. Open up the extensions manager by navigating to `Window / Extensions`. The extension is available as a THIRD PARTY extension. Search for `BECKHOFF BRIDGE`, and click the slider to Enable it. Once enabled, the extension will show up as an option in the top menu banner.
```
Comment (shanereetz) [inline exts/loupe.beckhoff_bridge/docs/README.md:9]:
```suggestion
This is the preferred method. Open up the extensions manager by navigating to `Window / Extensions`. The extension is available as a "Third Party" extension. Search for `Beckhoff Bridge`, and click the slider to Enable it. Once enabled, the extension will be available as an option in the top menu banner of Omniverse.
```

**Pull request merged**